wsagent/che-core-api-languageserver/src/main/java/org/eclipse/che/api/languageserver/registry/LanguageServerRegistryImpl.java

```diff
@@ -10,6 +10,17 @@
  *******************************************************************************/
 package org.eclipse.che.api.languageserver.registry;
 
+import static org.eclipse.che.api.languageserver.shared.ProjectLangugageKey.createProjectKey;
```

Comment:
Confusingly, the doc mentions the package groups in one order, then defines their order as the exact reverse of the one a couple of lines above.